packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -518,6 +518,33 @@ describe('<Tabs />', () => {
       expect(hasRightScrollButton(container)).to.equal(true);
     });
 
+    it('should have correct rotation when `orientation="vertical" & isRtl`', () => {
+      if (!isJSDOM) {
+        // testing in real browser result in different value of "transform"
+        // Chrome gives matrix
+        // Firefox gives rotate
+        // so testing in JSDOM is enough in this case
+        this.skip();
+      }
+      const { getByTestId } = render(
+        <Tabs value={0} variant="scrollable" orientation="vertical" scrollButtons>
+          <Tab />
+          <Tab />
+        </Tabs>,
+        {
+          wrapper: ({ children }) => (
+            <ThemeProvider theme={createTheme({ direction: 'rtl' })}>{children}</ThemeProvider>
+          ),
+        },
+      );
+      expect(getByTestId('KeyboardArrowRightIcon')).to.toHaveComputedStyle({
+        transform: 'rotate(-90deg)',
+      });
+      expect(getByTestId('KeyboardArrowLeftIcon')).to.toHaveComputedStyle({
+        transform: 'rotate(-90deg)',
+      });
+    });
+
```

Comment:
Let's do a visual regression test instead. That's what they're for in the end. JSDOM for testing visuals gives barely any confidence.

Reply:
That make senses, added regression.

Reply:
Nice, verified it's broken on `next`: https://codesandbox.io/s/vertical-tabs-rtl-ec308